pymedphys/cli/__init__.py

```diff
@@ -85,11 +85,6 @@ def run_logging_basic_config(args):
 
     if "level" not in logging_config:
         logging_config["level"] = logging.WARNING
-    else:
```

Comment:
The attribute search/validation may be needed.  DictConfig (not available until 3.2) states that it will validate and raise specific Errors.
basicConfig does not say it will raise those errors, nor does it say what it will do if provided a level it doesn't understand, either way.  I had assumed that if DictConfig would raise the errors, basicConfig would as well, but now I don't think that's a valid assumption.  I don't know if basicConfig is a wrapper around DictConfig, but now looking at the evolution of logging configuration, I'm not willing to assume they refactored to ensure DRY.
And separate from the validation, this change also resulted in loss of using the level specified in the config.toml file (e.g. ERROR, if someone really doesn't want much logging, or INFO if they didn't want to have to use the command line --verbose option to get INFO level logging...)

Reply:
In the end, I believe the main reason I needed this code back in so that the following line worked:

https://github.com/pymedphys/pymedphys/blob/fc9063d992ad93f65a3910c69b5e12a6660d62cf/pymedphys/cli/__init__.py#L105